src/go/networking/pkg/portproxy/server_test.go

```diff
@@ -54,7 +54,7 @@ func TestNewPortProxyUDP(t *testing.T) {
 		UpstreamAddress: testServerIP,
 		UDPBufferSize:   1024,
 	}
-	portProxy := portproxy.NewPortProxy(localListener, proxyConfig)
```

Comment:
This shuold use [`t.Context()`](https://cs.opensource.google/go/go/+/refs/tags/go1.24.5:src/testing/testing.go;l=1392) instead of `context.Background()`.  The same applies to other uses of `context.Background()` in tests (but let's just touch the ones you're changing here).

I don't know why `pkg.go.dev` seems to have lost the last half year of stdlib releases…